packages/apps/samples/bare-sample/project.json

```diff
@@ -1,17 +1,17 @@
 {
-  "sourceRoot": "packages/apps/samples/bare/src",
+  "sourceRoot": "packages/apps/samples/bare-sample/src",
   "projectType": "application",
   "targets": {
     "lint": {
       "executor": "@nrwl/linter:eslint",
       "options": {
         "format": "unix",
         "lintFilePatterns": [
-          "packages/apps/samples/bare/**/*.{ts,js,tsx,jsx}"
+          "packages/apps/samples/bare-sample/**/*.{ts,js,tsx,jsx}"
         ]
       },
       "outputs": [
-        "{options.outputFile}"
+        "{options.lintFilePatterns.0}"
```

Comment:
@wittjosiah i verified that nx actually supports `.0.` but i am not sure if this is necessary? idk ...

Reply:
What's the goal here? I think this would cache all the source code which seem like a bad idea to me